setup.cfg

```diff
@@ -1,7 +1,9 @@
+[bdist_wheel]
+universal = 1
+
 [metadata]
-name = tool
-version = attr: tool.version.__version__
-url = https://github.com/user-cont/tool
+name = upsint
+url = https://github.com/user-cont/upsint
```

Comment:
What about packit org, isn't it a better place for this?

Reply:
I don't care personally. We can vote if you want.

Reply:
I vote for packit! It can be taken as just another piece we develop to ease the developer's life.